### DISCUSSION TITLE
Locked from purchase

### DISCUSSION THREAD

**Discussion by haoRchen** [community/community#22976] [New to GitHub]
<p>I’m locked from adding free plugins despite updating my credit card recently. I’ve tried using Amex and Mastercard, still getting the same error. Why is my account locked from purchase? </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/6325a89c268a70691f71e1088472ba2788b0771c.png" data-download-href="/uploads/short-url/e95X4EmrEjX4L58Y4PAck8fAzEg.png?dl=1" title="Review your order · GitHub Marketplace - Brave 2019-10-29 20.51.10.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/19397242/181074101-b00aa90f-4240-4b6a-9808-40e3905a5d10.png" alt="Review your order · GitHub Marketplace - Brave 2019-10-29 20.51.10.png" data-base62-sha1="e95X4EmrEjX4L58Y4PAck8fAzEg" width="690" height="180" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/6325a89c268a70691f71e1088472ba2788b0771c_2_690x180.png, https://aws1.discourse-cdn.com/github/original/1X/6325a89c268a70691f71e1088472ba2788b0771c.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/6325a89c268a70691f71e1088472ba2788b0771c.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/6325a89c268a70691f71e1088472ba2788b0771c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Review your order · GitHub Marketplace - Brave 2019-10-29 20.51.10.png</span><span class="informations">1007×263 13.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/haorchen">@haorchen</a> </p>
<p>Unfortunately there’s not a lot people on the forum can do about this. Please contact support on <a href="https://support.github.com/contact" rel="nofollow noopener">https://support.github.com/contact</a></p>
